### PR TITLE
Blog post: exposing and resolving persistent identifiers on the web

### PR DESCRIPTION
**Pull request by rachelheaven**
Thank you for submitting a pull request for your new post. Please remember to assign the PR for review.

### Blog Post Title

* Exposing and resolving persistent identifiers on the web

### Link to rendered version
Please include a link to the rendered version of your blog post so it can be seen its intended display format,

* https://rachelheaven.github.io/BritishGeologicalSurvey.github.io/open%20data/persistent-identifiers/

### CODE REVIEW

File: _posts/2021-08-11-persistent-identifiers.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+title:  "Exposing and resolving persistent identifiers on the web"
+author: Rachel Heaven
+categories:
+  - Open Data
+tags:
+  - Linked Data
+  - RDF
+---
+
+# Exposing and resolving persistent identifiers on the web
```
Comment (Fuhji) [inline _posts/2021-08-11-persistent-identifiers.md]: This duplicate heading can be removed, as the page title is generated by the `title:` field on line 2

**Comment by Fuhji** [comment]
@pdbebgs Final post for review in this batch

File: _posts/2021-08-11-persistent-identifiers.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## What are persistent identifiers?
+
+Within our internal corporate databases we have well managed persistent identifiers for most of our important data items, in the form of foreign key columns in relational databases. These are either auto-incrementing integers or a short text code.  They uniquely identify a record, can be used to reference it from other database tables are not allowed to be changed. The codes are guaranteed to be unique within that set of data, but may not be unique across BGS's other data resources and certainly not unique outside BGS.
```
Comment (pdbebgs) [inline _posts/2021-08-11-persistent-identifiers.md]: Is there a missing 'and' - from other database tables AND are not allowed to be changed.

File: _posts/2021-08-11-persistent-identifiers.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+As part of the scheduled pipeline, all the triples are updated nightly to https://github.com/BritishGeologicalSurvey/vocabularies
+and the triples are re-loaded into our Apache Jena triplestore.  
+
+The linked data is exposed to the web using nginx and a Linked Data API written using the python based Falcon web framework. 
+
+## How do we redirect to a user friendly web page or application?
+
+
+The API handles content negotiation for different RDF (resource description framework) formats and uses [rdflib](https://github.com/RDFLib/rdflib) for serialisation.
```
Comment (pdbebgs) [inline _posts/2021-08-11-persistent-identifiers.md]: I got a little lost here.  I'm not quite sure why http://data.bgs.ac.uk/id/dataHolding/13606914.nt doesn't redirect but http://data.bgs.ac.uk/id/dataHolding/13606914 does redirect. Can you just read over again? I'm happy for you to conclude it's clear!

**Review by pdbebgs — COMMENTED** [review]
Great article Rachel. I've suggested one typo correct and made one comment. Thank you.

File: _posts/2021-08-11-persistent-identifiers.md (hunk 1 above)
Comment (rachelheaven) [reply _posts/2021-08-11-persistent-identifiers.md]: removed

File: _posts/2021-08-11-persistent-identifiers.md (hunk 2 above)
Comment (rachelheaven) [reply _posts/2021-08-11-persistent-identifiers.md]: fixed

File: _posts/2021-08-11-persistent-identifiers.md (hunk 3 above)
Comment (rachelheaven) [reply _posts/2021-08-11-persistent-identifiers.md]: Good point - not easy to spot the difference - I've added some clarification

**Comment by rachelheaven** [comment]
Sorry for the delay getting around to this - now ready for re-review !

**Comment by Fuhji** [comment]
Thanks @rachelheaven for the edits, @pdbebgs this is ready for your review now.

**Review by pdbebgs — APPROVED** [review]
Brilliantly written. Probably the clearest explanation I've ever read of URI publishing and linked data. I usually like to suggest some changes to demonstrate I've reviewed it properly, but really have got nothing to suggest!

**Comment by Fuhji** [comment]
@rachelheaven The post 'release' date is taken from the filenames, currently 2021-08-11. Please could you change the filenames to todays date i.e. 2022-02-22 (also is an interesting date to use!) and ill get this merged straight away

**Comment by rachelheaven** [comment]
have renamed

**Pull request merged**